core/src/com/unciv/models/UnitAction.kt

```diff
@@ -1,12 +1,11 @@
 package com.unciv.models
 
 data class UnitAction(
-        var type: UnitActionType,
-        var canAct: Boolean,
-        var title: String = type.value,
-        var isCurrentAction: Boolean = false,
-        var uncivSound: UncivSound = UncivSound.Click,
-        var action: (() -> Unit)? = null
+        val type: UnitActionType,
```

Comment:
Here's where I got rid of the `canAct` property.  Now `canAct` is the same as `action == null`